WorkflowConcurrency/Testing/TaskTesting.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if DEBUG
+
+    import Combine
+    import Workflow
+    import WorkflowTesting
+    import XCTest
+    @testable import WorkflowConcurrency
+
+    @available(macOS 10.15, *)
+    @available(iOS 13.0, *)
+    extension RenderTester {
+        /// Expect a `Task`s.
+        ///
+        /// `TaskWorkflow` is used to execute to `Task`s.
+        ///
+        /// - Parameters:
+        ///   - producingOutput: An output that should be returned when this worker is requested, if any.
+        ///   - key: Key to expect this `Workflow` to be rendered with.
+        public func expectTask<Value>(
```

Comment:
nit: are there other examples of `expectXYZ(xyz: ...)` in these utility methods? the ones i easily found all looked like `expect(xyz: ...)` and i'm inclined to keep this the same to preserve 'muscle memory' for people if that's the only existing naming convention.

Reply:
WorkflowConcurrency follows the same patterns as WorkflowCombine.  It's `expectPublisher` in `WorkflowCombineTesting`.  I'm fine with changing both of these.

Reply:
okay cool, i think that'd be a nice marginal improvement. good to keep logically equivalent APIs using the same naming conventions IMO.